StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -210,4 +211,11 @@ void CloseDeviceHandle(DeviceHandle& Handle)
     }
 }
 
+eErrorCode GetPhysicalDiskNumber(DeviceHandle Handle, U32& PhysicalDiskNumber)
+{
+    //! TODO Use Device path on Linux system instead PhysicalDiskNumber
+
+    return(eErrorCode::None);
```

Comment:
return eErrorCode::None because DriveEnumeratorAta/Scsi check None value with ErrorCode
 errorCode = GetPhysicalDiskNumber(deviceHandle, physicalDiskNumber);
    if (eErrorCode::None != errorCode)
    {
        //TODO: handle error
        return(nullptr);
    }